### PR TITLE
limit interval response size to 50

### PR DESCRIPTION
**Pull request by LiorAgnin**
subj

### CODE REVIEW

File: src/services/blockGraph.ts [hunk 1]
```diff
@@ -46,6 +46,6 @@ export default class BlockGraphService {
   async getPreviousBlock (duration: any) {
     const currentTime = dayjs()
     const previousTime = currentTime.subtract(duration).unix()
-    return this.getBlockFromTimestamp(previousTime)
+    return await this.getBlockFromTimestamp(previousTime)
```
Comment (leonprou) [inline src/services/blockGraph.ts:49]: you can delete the await here, as it's on the last line

File: src/services/token.ts [hunk 2]
```diff
@@ -105,14 +97,16 @@ export default class TokenService {
     )
   }
 
-  async getTokenPriceChangeInterval (tokenAddress: string, timeFrame = TimeFrame.MONTH, interval = Interval.HOUR) {
+  async getTokenPriceChangeInterval (tokenAddress: string, timeFrame = TimeFrame.MONTH) {
+    const MAX_RESULT_SIZE = 50
```
Comment (leonprou) [inline src/services/token.ts:101]: Can you move this to config?

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good, see the small comments :)

File: src/services/token.ts (hunk 2 above)
Comment (LiorAgnin) [reply src/services/token.ts]: Fixed @leonprou

File: src/services/blockGraph.ts (hunk 1 above)
Comment (LiorAgnin) [reply src/services/blockGraph.ts]: @leonprou Fixed

**Review by leonprou — COMMENTED** [review]
LGTM

**Pull request merged**